### PR TITLE
Fix SchemaBuilder fail on init request

### LINKED ISSUE #11: Schema Builder fails with DataStore null error
Reproduce by running `curl -i http://localhost:8080/init`

### PR DESCRIPTION
**Pull request by malinthar**
This resolves #11

### CODE REVIEW

File: src/main/resources/config-schema.yaml [hunk 1]
```diff
@@ -1,43 +1,8 @@
-hbase.zoookeepers: "localhost"
+hbase.zookeepers: "localhost"
 
-hbase.catalog: "dengDIDataStore"
+hbase.catalog: "weatherdata"
```
Comment (chanukaDissanayaka) [inline src/main/resources/config-schema.yaml]: Feature name as catalog

**Review by chanukaDissanayaka — CHANGES_REQUESTED** [review]
Confusion in catalog name and feature names

File: src/main/resources/config-schema.yaml (hunk 1 above)
Comment (malinthar) [reply src/main/resources/config-schema.yaml]: Catalog should be common for all the features right?

File: src/main/resources/config-schema.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 hbase.zookeepers: "localhost"
 
-hbase.catalog: "weatherdata"
+hbase.catalog: "dengDIDataStore"
```
Comment (malinthar) [inline src/main/resources/config-schema.yaml:3]: @chanukaDissanayaka 
This commit addresses your concern

File: src/main/resources/config-schema.yaml (hunk 1 above)
Comment (chanukaDissanayaka) [reply src/main/resources/config-schema.yaml]: As we discussed earlier?

**Review by chanukaDissanayaka — APPROVED** [review]

**Review by chanukaDissanayaka — APPROVED** [review]

**Pull request merged**